src/main/kotlin/no/nav/syfo/App.kt

```diff
@@ -56,6 +57,13 @@ fun main() {
             environment.redisSecret
         )
     )
+
+    val altinnClient = AltinnClient(
+        altinnWsUrl = environment.altinnWsUrl,
+        username = environment.altinnUsername,
+        password = environment.altinnPassword
+    )
```

Comment:
Denne brukes bra i ApiModule, så hva med å flytte den inn i ApiModule? Da blir testingen mer spesifikk.

Reply:
Vi har flyttet den ut fra ApiModule for å kunne mocke den fra PostDialogmoteApiV2Spek, der vi verifiserer at den er kalt.

Reply:
Se kommentaren min nedenfor vedr mocking, kortversjon: burde ikke mockeAltinnClient med mock, men heller mocke ekstern tjeneste(`IFlrReadOperations`).
Dette gir mer fullverdig test og gjør at man også får testet AltinnClient og SOAP-grensesnittet.

Reply:
Nå har det komme flere kommentarer vi må se på, men vi tenker kanskje å endre til soap-mock i en egen branch, så vi kan få merget dette i dag. Eller er det dumt?

Reply:
Det går fint det 👍